elliottlib/cli/find_bugs_cli.py

```diff
@@ -220,16 +220,52 @@ def _filter_bugs(bugs):  # returns a list of bugs that should be processed
         return
-    impetus_bugs = {}  # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
-    # @lmeyer: simple and stupid would still be keeping the logic in python, possibly with config flags for branched logic. until that logic becomes too ugly to keep in python, i suppose..
+
+    # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
+    impetus_bugs = {
+        "rpm": set(),
+        "image": set(),
+        "extras": set()
+    }
+
+    # @lmeyer: simple and stupid would still be keeping the logic in python,
+    # possibly with config flags for branched logic.
+    # until that logic becomes too ugly to keep in python, i suppose..
     if major_version < 4:  # for 3.x, all bugs should go to the rpm advisory
         impetus_bugs["rpm"] = set(bugs)
     else:  # for 4.x
-        # optional operators bugs should be swept to the "extras" advisory, while other bugs should be swept to "image" advisory.
-        # a way to identify operator-related bugs is by its "Component" value. temporarily hardcode here until we need to move it to ocp-build-data.
+        # sweep rpm cve trackers into "rpm" advisory
+        if mode == 'sweep' and cve_trackers:
+            rpm_bugs = bzutil.get_valid_rpm_cves(bugs)
+            green_prefix("RPM CVEs found: ")
+            click.echo([b.id for b in rpm_bugs])
+
+            # check if corresponding brew builds are attached to advisory
+            attached_builds = errata.get_advisory_nvrs(runtime.group_config.advisories["rpm"])
+            packages = attached_builds.keys()
+            failed = []
+            for bug, package_name in rpm_bugs.items():
+                if package_name not in packages:
+                    failed.append((bug.id, package_name))
+                    continue
+                impetus_bugs["rpm"] = {b for b in rpm_bugs}
+
+            if failed:
+                red_prefix("RPM CVE Warning: ")
+                click.echo("The following CVE tracker bugs were found but not attached, because no corresponding "
+                           "brew builds were found attached to the advisory. First attach builds and then rerun to "
+                           "attach "
+                           "the bugs")
+                click.echo(failed)
+
+        # optional operators bugs should be swept to the "extras" advisory
+        # a way to identify operator-related bugs is by its "Component" value.
+        # temporarily hardcode here until we need to move it to ocp-build-data.
         extra_components = {"Logging", "Service Brokers", "Metering Operator", "Node Feature Discovery Operator"}  # we will probably find more
         impetus_bugs["extras"] = {b for b in bugs if b.component in extra_components}
-        impetus_bugs["image"] = {b for b in bugs if b.component not in extra_components}
+
+        # all other bugs should go into "image" advisory
+        impetus_bugs["image"] = set(bugs) - impetus_bugs["extras"] - impetus_bugs["rpm"]
```

Comment:
bugs that go to the rpm advisory should be excluded as well.

Reply:
I'm not sure what I'm missing here.. 

bugs attached to image = all bugs - (bugs attached to extras) - (bugs attached to rpm)

Or are you saying the `impetus_bugs["rpm"]` would not have all rpm advisory bugs in some case?